app/src/main/java/org/gophillygo/app/utils/UserUtils.java

```diff
@@ -132,6 +131,6 @@ public static boolean isFlagPostingEnabled(Context context) {
     public static boolean isFabricEnabled(Context context) {
         String key = context.getString(R.string.general_preferences_fabric_logging_key);
         SharedPreferences sharedPreferences = PreferenceManager.getDefaultSharedPreferences(context);
-        return sharedPreferences.contains(key) && sharedPreferences.getBoolean(key, false);
+        return !sharedPreferences.contains(key) || !sharedPreferences.getBoolean(key, false);
```

Comment:
Given that the logic of this function has been inverted, we should rename it to `isFabricDisabled`

Reply:
Good call. (This was a suggested linter simplification.)